app/Console/Commands/UpdateOrchids.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Profile;
+use App\ProfileData;
+
+
+class UpdateOrchids extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'orchid:update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Loop through the profiles with orcid managed publications to sync the info';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $inc = 0;
+        $profiles = Profile::whereHas('data', function($query) { $query->where('type', 'information')->where('data->orc_id_managed', '1'); } )->get();
+
+        foreach($profiles as $profile) {
+            if ($profile->updateORCID()) {
+                $inc++;
+                $this->info("Updated ORCiD info for {$profile->full_name}");
+            }
+            else {
+                $this->error('An error has occurred updating ORCiD info for ' . $profile->full_name);
+            }
+        }
+
+        $this->info("Success: {$inc} profiles have been updated.");
```

Comment:
In addition to this, I think it would be a good idea to also log the overall result, so that when this gets run on a schedule we have something to look at to verify that it was actually run. It also might be a good idea to log any profiles that returned false when updating so that we can look into why they're not updating. See the laravel docs on logging: https://laravel.com/docs/8.x/logging#writing-log-messages